src/options/views/tab-settings/vm-sync.vue

```diff
@@ -22,9 +22,9 @@
       <label class="sync-server-url">
         <span v-text="i18n('labelSyncServerUrl')"></span>
         <input
-          type="text"
+          type="url"
```

Comment:
You also need to add `input[type=url]` in our [global css](https://github.com/violentmonkey/violentmonkey/blob/master/src/common/ui/style/style.css) in three places where `input[type=text]` is currently used.

Reply:
Done.